src/calculateRentalCost.js

```diff
@@ -29,6 +29,27 @@
 
 function calculateRentalCost(days) {
   // write code here
+  const cost = 40;
+  const total = cost * days;
+  const largeDiscount = 50;
+  const smallDiscount = 20;
+  const noDiscount = 2;
+  const largeDiscountReq = 7;
```

Comment:
what is Req? It's not necessary to add this I guess

Reply:
It's a requirement.